release-notes/april-2025.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: april-2025
+title: April 2025
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+tags:
+  - Release notes
+hide_table_of_contents: false
+date: 2025-04-30T10:00
+---
+
+# Semgrep release notes for April 2025
+
+<!-- Remember to update latest endpoint -->
+<!-- Remember to update index page -->
+
+The following updates were made to Semgrep in April 2025.
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+### Changed
+
+### Fixed
+
+## 💻 Semgrep Code
+
+### Added
+
+### Changed
+
+### Fixed
+
+## ⛓️ Semgrep Supply Chain
+
+### Added
+
+- [SBOM exports through the Semgrep API](https://semgrep.dev/api/v1/docs/#tag/SupplyChainService/operation/semgrep_app.products.sca.handlers.sbom.openapi_create_sbom_export) is now generally available.
```

Comment:
```suggestion
- [SBOM export through the Semgrep API](https://semgrep.dev/api/v1/docs/#tag/SupplyChainService/operation/semgrep_app.products.sca.handlers.sbom.openapi_create_sbom_export) is now generally available.
```
Subject-verb agreement correction: _SBOM **export** ... is now generally available._